setup.cfg

```diff
@@ -89,3 +89,8 @@ addopts =
   --cov-report=term
   --cov-report=xml
   --junitxml=pytest.xml
+
+[pydocstyle]
+convention=numpy
+match_dir= ^(?!(docs|\.)).*
+match= (?!(test_|setup)).*\.py
```

Comment:
Again, I don't think the `setup` is need as this should not be included in the test path. As for `test_` these seem to pass anyway, so I don't think this line is required.